### PR TITLE
use certificates for RDS connection

### PR DESCRIPTION
**Pull request by asmega**
- also we seem to be bundling test and dev in production. this PR will fix this

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ stop:
 	docker-compose down -v
 
 build: stop
-	docker-compose build --build-arg BUNDLE_FLAGS=''
+	docker-compose build --build-arg BUNDLE_FLAGS='--jobs 2 --no-cache'
```
Comment (cob16) [inline Makefile]: wow didn't know about `--jobs`!

File: deploy/fb-publisher-chart/templates/deployment.yaml [hunk 2]
```diff
@@ -35,8 +35,11 @@ spec:
           - configMapRef:
               name: fb-publisher-env-{{ .Values.environmentName }}
         env:
-          # external secrets
-          # defined in secrets.yml
+          - name: DATABASE_URL
+            valueFrom:
+              secretKeyRef:
+                name: rds-instance-formbuilder-publisher-{{ .Values.environmentName }}
+                key: url
```
Comment (cob16) [inline deploy/fb-publisher-chart/templates/deployment.yaml]: if you put this in the config map would not have to repeat yourself?

File: docker/worker/Dockerfile [hunk 3]
```diff
@@ -4,17 +4,27 @@ FROM ministryofjustice/ruby:2.5.1
...
+RUN bundle install ${BUNDLE_FLAGS}
 
-COPY . .
+ADD --chown=deploy:deploy https://s3.amazonaws.com/rds-downloads/rds-ca-2019-root.pem ./rds-ca-2019-root.pem
+ADD --chown=deploy:deploy https://s3.amazonaws.com/rds-downloads/rds-ca-2015-root.pem ./rds-ca-2015-root.pem
+RUN cat ./rds-ca-2019-root.pem > ./rds-ca-bundle-root.crt
+RUN cat ./rds-ca-2015-root.pem >> ./rds-ca-bundle-root.crt
+RUN chown deploy:deploy ./rds-ca-bundle-root.crt
+
+COPY --chown=deploy:deploy . .
```
Comment (cob16) [inline docker/worker/Dockerfile]: nice to see :+1:

File: deploy/fb-publisher-chart/templates/deployment.yaml (hunk 2 above)
Comment (asmega) [reply deploy/fb-publisher-chart/templates/deployment.yaml]: how do i add to config map if it's from a secret?

**Pull request merged**